spec/runtime/with_clean_env_spec.rb

```diff
@@ -2,107 +2,104 @@
 require "spec_helper"
 
 describe "Bundler.with_env helpers" do
-  shared_examples_for "Bundler.with_*_env" do
-    it "should reset and restore the environment" do
-      gem_path = ENV["GEM_PATH"]
-      path = ENV["PATH"]
+  describe "Bundler.original_env" do
+    it "should return the PATH present before bundle was activated" do
+      gemfile ""
+      bundle "install --path vendor/bundle"
```

Comment:
These two lines seem to be present in each of the `it` blocks for `describe "Bundler.original_env"` and `describe "Bundler.clean_env"`:
```ruby
      gemfile ""
      bundle "install --path vendor/bundle"
```

They could be extracted into a single `before` block.